tunder/lib/src/database/operations/postgres/postgres_query_operation.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:tunder/database.dart';
+import 'package:tunder/src/database/operations/contracts/query_operation.dart';
+import 'package:tunder/src/database/operations/postgres/mixins/where_compiler.dart';
+
+class PostgresQueryOperation with WhereCompiler implements QueryOperation {
+  late final Query query;
+
+  @override
+  Future<List<MappedRow>> process(Query query) async {
+    return DB.query(toSql(query));
```

Comment:
Could write this a few others single line statements with the arrow notation, like:
```dart
Future<List<MappedRow>> process(Query query) => DB.query(toSql(query));
```

Reply:
Yeah, that's just a force of habit. 😅